src/main/java/com/conveyal/gtfs/validator/ReversedTripValidator.java

```diff
@@ -50,7 +50,8 @@ public boolean validate(Feed feed, boolean repair) {
             if (trip.shape_id == null) {
                 isValid = false;
                 if (missingShapeErrorCount < errorLimit) {
-                    feed.errors.add(new MissingShapeError(trip));
+                    // FIXME store MissingShape errors
+                    // feed.errors.add(new MissingShapeError(trip));
```

Comment:
What's with all these changes related to shape errors?

Reply:
When working on this, I discovered that the `Feed#errors` property was being used by some other validators (including `ReversedTripValidator`) to store errors that were never making it into the database. We have an open issue (#130) that somewhat covers this already, although I'm not sure if we have a good plan of attack for how to restore these checks.

Reply:
Is that an MTC-related need?

Reply:
Good one.